### PR TITLE
Update attack-surface-reduction-rules-reference.md

### PR DESCRIPTION
**Pull request by msbemba**
Onenote shows  few hits on our ASR Audit

Added the same to the list.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10478

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ Attackers might attempt to use Office apps to migrate malicious code into other
 
 There are no known legitimate business purposes for using code injection.
 
-This rule applies to Word, Excel, and PowerPoint.
+This rule applies to Word, Excel, Onenote, and PowerPoint.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md]:
```suggestion
This rule applies to Word, Excel, OneNote, and PowerPoint.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by bweston-8 — APPROVED** [review]
Looks good. Approved

**Pull request merged**